pkg/utils/utils.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+func GetClientset() (*kubernetes.Clientset, error) {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, err
+	}
+
+	return clientset, nil
+}
```

Comment:
This is used only once in the `main.go`, do we need it in a separate package? Couldn't it stay as unexported function in `main.go`? I don't like generic `utils` packages.

Reply:
Yep, fine by me.